src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Heavy.ttf");
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif ;
+}
+
+.form_field {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+}
+
+.icon_small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.icon_big {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.field {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+}
+
+.field_big {
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
```

Comment:
Due to checklist no need to create `field_big` class, you can move all styles under general `field `class and modify styles for small field in `field_small` class
![image](https://user-images.githubusercontent.com/80700247/232210177-cb5ca515-ae55-4194-99b8-9f93dfb6d00f.png)

Fix everywhere